boot/zephyr/Kconfig

```diff
@@ -556,6 +556,12 @@ config BOOT_ERASE_PROGRESSIVELY
 	 on some hardware that has long erase times, to prevent long wait
 	 times at the beginning of the DFU process.
 
+config BOOT_MGMT_ECHO
+	bool "Enable echo command"
+	default "n"
```

Comment:
```suggestion
```
For any Kconfig property we always get implicate `default n` if not gives it y explicitly.

Reply:
Thanks for clarifying. I've updated the file to your comment.